druid/examples/widget_gallery.rs

```diff
@@ -0,0 +1,386 @@
+            include_str!("./assets/tiger.svg")
+                .parse::<SvgData>()
+                .unwrap(),
+        ),
+        "Svg",
+    );
+
+    #[cfg(target_arch = "wasm32")]
+    let svg_example = label_widget(Label::new("no SVG on wasm (yet)").center(), "Svg");
+
+    Scroll::new(
+        SquaresGrid::new()
+            .with_cell_size(Size::new(200.0, 240.0))
+            .with_spacing(20.0)
+            .with_child(label_widget(
+                Label::new(|data: &AppData, _: &_| data.label_data.clone()),
+                "Label",
+            ))
+            .with_child(label_widget(
+                Flex::column()
+                    .with_child(
+                        Button::new("Click me!")
+                            .on_click(|_, data: &mut AppData, _: &_| data.clicked_count += 1),
+                    )
+                    .with_spacer(4.0)
+                    .with_child(Label::new(|data: &AppData, _: &_| {
+                        format!("Clicked {} times!", data.clicked_count)
+                    })),
+                "Button",
+            ))
+            .with_child(label_widget(
+                Checkbox::new("Check me!").lens(AppData::checkbox_data),
+                "Checkbox",
+            ))
+            .with_child(label_widget(
+                List::new(|| {
+                    Label::new(|data: &String, _: &_| format!("List item: {}", data))
+                        .center()
+                        .background(Color::hlc(230.0, 50.0, 50.0))
+                        .fix_height(40.0)
+                        .expand_width()
+                })
+                .lens(AppData::list_items),
+                "List",
+            ))
+            .with_child(label_widget(
+                Flex::column()
+                    .with_child(ProgressBar::new().lens(AppData::progressbar))
+                    .with_spacer(4.0)
+                    .with_child(Label::new(|data: &AppData, _: &_| {
+                        format!("{:.1}%", data.progressbar * 100.0)
+                    }))
+                    .with_spacer(4.0)
+                    .with_child(
+                        Flex::row()
+                            .with_child(Button::new("<<").on_click(|_, data: &mut AppData, _| {
+                                data.progressbar = (data.progressbar - 0.05).max(0.0);
+                            }))
+                            .with_spacer(4.0)
+                            .with_child(Button::new(">>").on_click(|_, data: &mut AppData, _| {
+                                data.progressbar = (data.progressbar + 0.05).min(1.0);
+                            })),
+                    ),
+                "ProgressBar",
+            ))
+            // The image example here uses hard-coded literal image data included in the binary.
+            // You may also want to load an image at runtime using a crate like `image`.
+            .with_child(label_widget(
+                Painter::new(|ctx, _, _| {
```

Comment:
I would move this to a separate function or something just on the pretext of trying to keep this sane.